tools/alara_output_processing/alara_output_processing.py

```diff
@@ -1,11 +1,21 @@
 import pandas as pd
 import argparse
+import operator
 from warnings import warn
 from csv import DictReader
 from numpy import array
 
 # ---------- General Utility Methods ----------
 
+OPS = {
+    '>'  : operator.gt,
+    '<'  : operator.lt,
+    '>=' : operator.ge,
+    '<=' : operator.le,
+    '==' : operator.eq,
+    '!=' : operator.ne
```

Comment:
Given the precision of floating point operations I wonder if we need any of these that include equality? Limiting it to just `<` and `>` might make it all a little simpler, too.